src/main/menus/app.js

```diff
@@ -158,31 +121,11 @@ function createTemplate(config) {
     accelerator: process.platform === 'darwin' ? 'Ctrl+Cmd+F' : 'F11',
   }, separatorItem, {
     label: 'Actual Size',
-    accelerator: 'CmdOrCtrl+0',
-    click() {
-      const focused = webContents.getFocusedWebContents();
-      focused.setZoomLevel(1);
-    },
+    role: 'resetZoom',
   }, {
-    label: 'Zoom In',
-    accelerator: 'CmdOrCtrl+SHIFT+=',
-    click() {
-      const focused = webContents.getFocusedWebContents();
-      const level = focused.getZoomLevel();
-      if (level <= 3) {
-        focused.setZoomLevel(level + ZOOM_DIFFERENTIAL);
-      }
-    },
+    role: 'zoomIn',
```

Comment:
Zooming seems to be broken when the focus isn't on the `BrowserView`. I've fixed a few of those cases in my latest PR, but I'm not sure what's going to happen here if you were to click on a tab and then click Zoom, since the in-focus window is the tab bar, not the MattemostView.

Reply:
yeah, that was mi experience the first time I did that, but while trying several things this was working properly. I could switch tabs and zoom, and it was the content that was zoomed not the tab. Did you manage to break it? if so, do you have repro steps? I tried several ways and it was always working for me

Reply:
on macOS catalina, if you go to another window and then come back and then use the zoom in keyboard shortcut, it will zoom the tabs in rather than the mattermost view. I think it could be fixed by ensuring focus goes to the mattermost view when refocusing the window rather than the main window web contents.